hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1236,10 +1236,17 @@
       name: chip_sw_alert_handler_lpg
       desc: '''Verify alert's low_power_group(lpg).
 
-            Choose an always-on block and send alerts during deep sleep and normal sleep mode,
-            check the alert_senders can recover after system comes out of both sleep modes.
-            Repeat the above sequence for multiple rounds with randomly chosen deep or normal sleep
-            mode for each round.
+            Three methods to check LPG:
+            - Choose an always-on or any block (for normal sleep mode only) and send alerts during
+              deep sleep and normal sleep mode, check the alert_senders can recover after system
+              comes out of both sleep modes.
+              Repeat the above sequence for multiple rounds with randomly chosen deep or normal
+              sleep mode for each round.
+            - Configure clkmgr to randomly turn off one of the IP's clock and check alert_handler
+              won't trigger a ping timeout error on that block.
+            - [TODO: this sequence can be time sensetive] Right when alert_handler is about to send
+              a ping test to an IP, configure rstmgr to manually reset that IP. Check alert_handler
```

Comment:
This is possible only for the reset-able IPs (per rstmgr SW_RST_CTRL_N_*). I suppose we also have a test that checks the absence of ping failures during deep or normal sleep?

Reply:
That is a good point, yes! This only apply to resetable IPs.
And totally agree - we plan to add another entry to make sure that the ping timer won't send spurious timeout errors when crossing sleep modes boundaries.

I am planning to bring up this topic in DV meeting and see if people have any other concerns/suggestions for this LPG testplan.